### ISSUE TITLE
[Wave 8] [Ideal Nav] Update Account Settings pages to use the new card styles

### ISSUE THREAD

**Issue by trjExpensify** [Expensify/App#32952]
Coming from [this convo](https://expensify.slack.com/archives/C05DWUDHVK7/p1702402241333569). 

The account settings pages have been moved to the main pane as part of the ideal nav skeleton project, but we didn't update them to the card style design to decouple the implementation from the navigational change. With the feature branch on the cusp of being merged, we're ready to address these changes. 

@dannymcclain has gone ahead and redesigned them, so the solution mocks can be found [here](https://github.com/Expensify/App/issues/32952#issuecomment-1919504860). 

As per @mountiny's [suggestion](https://github.com/Expensify/App/issues/32952#issuecomment-1919522655), we'll take this page by page instead of one giant PR for them all. Starting with the profile page which will add the card style to rinse/repeat for the others. 

CC: @shawnborton @dannymcclain @JmillsExpensify

**Comment by dannymcclain** [comment]
Just for supreme clarity, that would be these pages, correct?

**Account**
- Profile
- Wallet
- Share code
- Preferences
- Security
- Sign out (not really a page I guess)

**General**
- Help (also not really a page, right?)
- About

![CleanShot 2023-12-13 at 07 46 04@2x](https://github.com/Expensify/App/assets/7516624/93b73468-3d80-45a9-9db7-b3f7626b83dd)

**Comment by trjExpensify** [comment]
Yah mon, those ones!

**Comment by shawnborton** [comment]
Also if we need to rejig some of the animations for these (or get new ones) we totally can. I'm guessing now that the cards are going to be a bit wider for things like Preferences, we might need to adjust that animation so the arms are way longer?

**Comment by dannymcclain** [comment]
Not overdue Melvin—just haven't gotten to it yet!

**Comment by dannymcclain** [comment]
Sorry Melv, I think I can get to this soon!

**Comment by trjExpensify** [comment]
Skelly is looking pretty good on the latest adhoc build of the ideal nav branch, but it's not on prod yet Melv so this is still technically held.

**Comment by JmillsExpensify** [comment]
Woo, looking forward to this! @dannymcclain did you by chance land on the final mocks for this page? Asking because wow the v1 we did for ideal nav is really starting to show it's age. The "card" style looks so so much better.

**Comment by shawnborton** [comment]
Oh heck yeah brother, these are looking good already!

**Comment by dannymcclain** [comment]
@shawnborton one thing I'd like to talk to you and @dubielzyk-expensify about is how our edge to edge option rows work with these new cards. In the past, we've used 20px of left/right padding for everything so them going edge-to-edge wasn't ever an issue.

But now, with varying padding based on viewport (20px mobile, 32px desktop), do they get increased padding on desktop? Or do they stay the same across all viewport sizes?

![image](https://github.com/Expensify/App/assets/7516624/62769b95-ccc4-4f3e-ad58-7f395efa648e)

TBH them staying the same across viewports ☝️ doesn't look pad with the way they're currently styled.

I think the issue that arises is more with the push-to-page inputs, which definitely stand out as being not-aligned when they keep their 20px of padding the same on desktop.

![image](https://github.com/Expensify/App/assets/7516624/c3111b88-bb5d-4474-ac9a-db7b558c2567)

Feel free to take this convo to Slack or a video chat if that makes more sense. I'm just a little stumped as to how to handle these.

**Comment by dannymcclain** [comment]
Ok that would look like this. I think this is probably the best approach until we revisit the edge-to-edge-ness of all these components 😈

![image](https://github.com/Expensify/App/assets/7516624/b763afa4-9beb-4421-bfd4-57b6dc2ce1b8)

![image](https://github.com/Expensify/App/assets/7516624/b5de5ce2-bc68-4372-a1e4-fb7f33964d47)

**Comment by shawnborton** [comment]
Nice, I agree - let's go with that for now until we revisit all of these. Can't wait until they all look more like the subnav 😈

**Comment by dubielzyk-expensify** [comment]
Yeah. Let's get to these option row components soon 😅

**Comment by dannymcclain** [comment]
[Posted screens in Slack for feedback!](https://expensify.slack.com/archives/C03TME82F/p1706118342442709)

**Comment by dannymcclain** [comment]
[More updates posted in Slack.](https://expensify.slack.com/archives/C03TME82F/p1706549218475379?thread_ts=1706118342.442709&cid=C03TME82F) Getting close I think.

**Comment by JmillsExpensify** [comment]
Went ahead and removed the hold since we're going to start prioritizing this.

**Comment by mountiny** [comment]
In touch with SWM and they might get someone to work on this tomorrow using the ideal-nav as a base if it wont be merged to main yet

Just for ease of tracking I propose we split the implementation up to issue per the screen and then we start with the profile page implementation while other pages will be on hold for it (since lots of the card design will be shared).

Once that one page is done and merged, SWM can quickly follow up with the remainder of the pages in parallel. Thoughts?

**Comment by JmillsExpensify** [comment]
Agree with going page by page.

**Comment by trjExpensify** [comment]
Yeah, let's have a PR per page 👍

**Comment by trjExpensify** [comment]
Obviously noting that there might be a bit of time inbetween where we have some pages designed a little different, but I think that's okay to avoid a jumbo PR for all of them. 👍

**Comment by mountiny** [comment]
Yep, I think thats fine and should be resolved in matter of days

**Comment by trjExpensify** [comment]
Yeah, sounds good to me. Updated the OP, now we're ready to rock!

**Comment by mountiny** [comment]
I will create the issues for this tomorrow and add it to the project

**Comment by kosmydel** [comment]
Hey, I'm from Software Mansion, and I will be responsible for this issue. 

Can we consider Figma as the primary source of truth for design changes? If there are any modifications to the design, we should update them in Figma. This approach will simplify our work and eliminate confusion. Thanks!

**Comment by filip-solecki** [comment]
Hi! I'm Filip from Software Mansion and I will work on this issue too!

**Comment by kosmydel** [comment]
Thanks @mountiny! Could you also create a separate issue for handling the icon in the `HeaderWithBackButton` for all pages?

**Comment by mountiny** [comment]
@kosmydel Issues created, first starting with these two:
- https://github.com/Expensify/App/issues/35609
- https://github.com/Expensify/App/issues/35603

And the rest is on hold for those two just so we can then reuse the patterns established from them:
- https://github.com/Expensify/App/issues/35604
- https://github.com/Expensify/App/issues/35605
- https://github.com/Expensify/App/issues/35606
- https://github.com/Expensify/App/issues/35607
- https://github.com/Expensify/App/issues/35608

**Comment by trjExpensify** [comment]
PRs for https://github.com/Expensify/App/issues/35609 & https://github.com/Expensify/App/issues/35603 in review.

**Comment by trjExpensify** [comment]
Going to close this overall tracker as we have the individual issues for payments etc.
